apps/publisher/modules/page-decorators.js

```diff
@@ -233,6 +233,14 @@ var pageDecorators = {};
             }
         }
     };
+    pageDecorators.taxonomyAvailability = function(ctx, page, utils) {
+        var app = require('rxt').app;
```

Comment:
Please change this to:

page.taxonomyAvailability  = app.isTaxonomyEnabled() ;